src/test/java/reporter/CustomReporter.java

```diff
@@ -0,0 +1,117 @@
+package reporter;
```

Comment:
please move the package to src/main/java/common/reporter

Reply:
Done